### DISCUSSION TITLE
v2.6.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3899] [分享 / Share]
✨新增
- `快速收起评论区` 添加了平滑滚动效果. (PR #3857 by [Mike Lei](https://github.com/mikelei8291))
- `删除广告` 新增对搜索页广告的的屏蔽. (#3863)
- `显示视频投稿时间` 适配新版播放页. (#3860, PR #3880 by [wisokey](https://github.com/wisokey))
- `网址参数清理` 新增 `buvid`, `mid`, `up_id` 的清理. (#3896)
- `选集区域优化` 支持折叠合集下的分组. (#3764)

🐛修复
- 重新调整了 `自定义顶栏` 的历史弹窗过滤器, 改为只能选择 `全部` 或者单一类型 (#2729), 解决专栏加载不出来的问题 (#3676).
- 修复 `下载字幕` 在 3.x 播放器检测不到语言选项的问题. (#3873)
- 修复旧版播放页中高能进度条未跟随主题颜色. (#3830)
- 修复 `选集区域优化` 临时收起时, 多行标题出现布局问题. (#3841)
- 修复 `禁用特殊弹幕样式` 失效. (#3886)
- 修复 `删除视频弹窗` 失效. (#3853)
- 修复 `自定义顶栏` 开启后原版顶栏高度异常. (#3833) (应该还需要再优化下, 现在还会有点小空隙, 不过没之前那么离谱了)
- `隐藏视频标题层` 补充 `视频` 标签. (#3839)

元旦快乐~

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.0'>v2.6.0</a>.</em>

**Comment by Geeyun-JY3** [comment]
接着 https://github.com/the1812/Bilibili-Evolved/discussions/3764#discussioncomment-4011316 继续反馈：

由于目前 `选集区域优化` 的 `展开选集列表` 会展开合集下的所有分组，但分组右上角展开 / 收起操作的按钮在未做任何操作的情况下仍是未展开的状态（即「v」状），这就导致首次折叠合集中非当前视频所在分组需要点按两次上述按钮。

是否还能优化？如让按钮符合分组实际展开 / 收起状态（即让其他分组上述按钮在分组实际已展开时呈倒「v」状），或和破站默认逻辑一样[只强制展开当前视频所在分组](https://github.com/the1812/Bilibili-Evolved/discussions/3764#discussioncomment-4061612)。

**Reply by the1812** [reply]
可以只展开当前视频所在分组

**Reply by Geeyun-JY3** [reply]
已经另开了一个 issue 力：#4040
